src/App.js

```diff
@@ -11,12 +11,26 @@ export default function App() {
   return (
     <Router>
       <div className='App'>
+        {/* <ul>
+          <li>
+            <Link to="/List">List</Link>
+          </li>
+          <li>
+            <Link to="/Detail">Detail</Link>
+          </li> 
+        </ul> */}
         <Switch>
-          <Route>
+          <Route exact path="/Detail/:id"> 
+            <BeanieDetail />
+            {/* this route should point to a particular beanie baby by id and render that specific BeanieDetail page */}
+          </Route>
+          <Route exact path="/:page">
+            <BeaniesPage />
```

Comment:
This route is doing the same thing as your `/` route below.  You probably don't need both.  Everything still works, though!